platform_neuro_flow_api/api.py

```diff
@@ -72,6 +78,27 @@
 
 logger = logging.getLogger(__name__)
 
+CONFIG: web.AppKey[Config] = web.AppKey("config", None)
```

Comment:
Please use the real type instead of None, e.g.
```suggestion
CONFIG: web.AppKey[Config] = web.AppKey("config", Config)
```